src/gc/unix/gcenv.unix.cpp

```diff
@@ -741,6 +741,12 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
     return g_currentProcessCpuCount;
 }
 
+uint32_t GCToOSInterface::GetCurrentProcessCpuLimit()
+{
+    uint32_t cpuLimit, cpuCount = GCToOSInterface::GetCurrentProcessCpuCount();
```

Comment:
A nit - can you please structure the code the same way as in the gcenv.os.cpp? It seems easier to read than this condensed version.